ports/comms-ublox/vcpkg.json

```
@@ -1,6 +1,7 @@
 {
   "name": "comms-ublox",
   "version-string": "0.20.0",
+  "port-version": 2,
```

```suggestion
  "port-version": 1,
```

Port versions start at 0 (omitting `"port-versions"` defaults to 0) so this is technically port version 1.
Same for the other ports.

---

oh. didn't notice. thanks for the hint.